examples/rivets/js/components/todos.js

```diff
@@ -0,0 +1,41 @@
+/*global rivets, TodoStore */
+(function (rivets, TodoStore) {
+	'use strict';
+
+	// Controller for the todos app.
+	var TodosController = function () {
+		this.addTodo = this.addTodo.bind(this);
+		this.todos = TodoStore;
+		this.newTodo = '';
+	};
+
+	// Adds a new todo item.
+	TodosController.prototype.addTodo = function (ev) {
+		ev.preventDefault();
+
+		if (this.newTodo && this.newTodo.trim()) {
```

Comment:
Can you make this explicit by referring to the length? Makes it easier to follow, in particular for those not intimately familiar with the JS truth table.